### ISSUE TITLE
Self signed certificate in certificate chain

### ISSUE THREAD

**Issue by hadson172** [raineorshine/npm-check-updates#694]
- [X] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [X] I am using the latest version of `npm-check-updates`
- [X] I am using `node >= 10.17`

---------------------------
We are using internal repository (npm have access to all the stuff) but when I am trying to run ncu or ncu -u i'm gettin an error:  (is it possible to disable strict-ssl? like in npm ?)

```
[--------------------] 0/37 0%/usr/local/lib/node_modules/npm-check-updates/lib/npm-check-updates.js:427
  throw err
  ^

FetchError: request to [internalrepository/internal-library] failed, reason: self signed certificate in certificate chain
    at ClientRequest.<anonymous> (/usr/local/lib/node_modules/npm-check-updates/node_modules/minipass-fetch/lib/index.js:97:14)
    at ClientRequest.emit (events.js:314:20)
    at TLSSocket.socketErrorListener (_http_client.js:469:9)
    at TLSSocket.emit (events.js:326:22)
    at emitErrorNT (internal/streams/destroy.js:100:8)
    at emitErrorCloseNT (internal/streams/destroy.js:68:3)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  code: 'SELF_SIGNED_CERT_IN_CHAIN',
  errno: 'SELF_SIGNED_CERT_IN_CHAIN',
  type: 'system'
}
```

**Comment by SteffenAnders** [comment]
bump

**Comment by raineorshine** [comment]
This is a [bug in pacote](https://github.com/npm/pacote/issues/43) as far as we can tell. The issue has not yet received a reply.

`npm-check-updates` [loads your npm config](https://github.com/raineorshine/npm-check-updates/blob/master/lib/package-managers/npm.js#L13-L24) and [passes it to pacote](https://github.com/raineorshine/npm-check-updates/blob/master/lib/package-managers/npm.js#L79), which should handle your config the same as npm.

Duplicate: #636

**Comment by isaacs** [comment]
Does your request to `fontawesome` use a `ca` or `cafile` config?  That appears to be the problem, I don't see `minipass-fetch` doing anything with that when it creates the underlying node http/https request.

**Comment by raineorshine** [comment]
@hadson172 @SteffenAnders @motabass Potential fix made in #713 and published in `v7.0.3-alpha.1`. Will you try it out and let me know if it fixes the problem for you? 

```sh
npm install -g npm-check-updates@next
```

**Comment by hadson172** [comment]
For me works, getting timeout on small project (30 sec is to little) but this no problem i think. Thanks for the fix.
